docs/destinations/forwarding-events/google-tag-manager-server-side/amplitude-tag-for-gtm-ss/amplitude-tag-configuration/index.md

```diff
@@ -4,6 +4,12 @@ date: "2021-11-24"
 sidebar_position: 100
 ---
 
+:::tip Populating the Amplitude Session ID
+
+In order to populate the Session ID in Amplitude so that your events will be stitched into sessions correctly, your Snowplow events need to have the [`client_session` context entity](/docs/collecting-data/collecting-from-own-applications/javascript-trackers/javascript-tracker/javascript-tracker-v3/tracker-setup/initialization-options/index.md#session-context) attached. Then the Amplitude Tag will automatically populate the Amplitude Session ID based on the `firstEventTimestamp` of the session the event belongs to.
```

Comment:
```suggestion
In order to populate the Session ID in Amplitude so that your events are stitched into sessions correctly, your Snowplow events need to have the [`client_session` context entity](/docs/collecting-data/collecting-from-own-applications/javascript-trackers/javascript-tracker/javascript-tracker-v3/tracker-setup/initialization-options/index.md#session-context) attached. Then the Amplitude Tag will automatically populate the Amplitude Session ID based on the `firstEventTimestamp` of the session the event belongs to.
```